docs/embedding/sdk/introduction.md

```diff
@@ -20,10 +20,10 @@ To give you and idea of what's possible with the SDK, we've put together example
 
 ## Embedded analytics SDK prerequisites
 
-- React application. The SDK is tested to work with React 18 or higher, though it may work with earlier versions.
+- React application. The SDK is tested to work with React 18 and Next.js 14, though it may work with earlier versions.
```

Comment:
This may sound like we only tested it with next and not other frameworks, maybe we should clarify that since we only tested react 18, "the sdk therefore not guaranteed to work with next 15, which requires react 19"
I don't want to say that it doesn't work as it _may_ work, and it may start or stop working depending on what they do on the next and react side

Reply:
I will remove the mention of Next here and just leave it on the specific page for Next.js. WDYT?